packages/components/src/Layout/Semantics/Semantics.test.tsx

```diff
@@ -152,4 +152,16 @@ describe('Semantics', () => {
       height: 'fit-content',
     })
   })
+
+  test('Aside can use t-shirt sized for its width.', () => {
+    renderWithTheme(
+      <Layout hasAside>
+        <Aside width="medium">Aside content</Aside>
+        <Section as="main">Main content</Section>
+      </Layout>
```

Comment:
Example can be narrowed down to just the `Aside` itself.

```suggestion
      <Aside width="medium">Aside content</Aside>
```